jujugui/static/gui/src/app/components/user-profile/model-list/model-list.js

```diff
@@ -209,19 +209,26 @@ YUI.add('user-profile-model-list', function() {
           </div>
         );
       }
-      const list = this.state.modelList;
-      if (!list || list.length === 0) {
-        return null;
-      }
-      const rows = list.map(this._generateRow);
       const props = this.props;
+      const acl = props.acl;
       let createNewButton;
-      if (props.canCreateNew) {
+      if (acl && acl.canAddModels()) {
```

Comment:
The think that I am not sure about is the defensive if (acl && ...
This seems to suggest that acl could be undefined or null, which should never be the case in this code path, so that's just confusing. If it is for testing purposes, let's fix the tests.